### PR TITLE
Option to report unanalysed templates

### PR DESCRIPTION
**Pull request by MartinMystikJonas**
I was not able to find out how to test it because rule tests forces analysedPaths to empty array

### CODE REVIEW

**Comment by MartinMystikJonas** [comment]
Tested it on real world app and seems it works like a charm

File: src/Rule/LatteTemplatesRule.php [hunk 1]
```diff
@@ -84,6 +90,14 @@ public function processNode(Node $collectedDataNode, Scope $scope): array
                 $this->analyseTemplates($templates, $scope, $errors);
             }
         }
+
+        foreach ($this->analysedTemplatesRegistry->getReportedUnanalysedTemplates() as $templatePath) {
+            $errors[] = RuleErrorBuilder::message('Latte template ' . pathinfo($templatePath, PATHINFO_BASENAME) . ' was not analysed.')
+                ->file($templatePath)
+                ->line(1)
```
Comment (lulco) [inline src/Rule/LatteTemplatesRule.php]: maybe -1 or null or nothing?

**Review by lulco — APPROVED** [review]

File: src/Rule/LatteTemplatesRule.php (hunk 1 above)
Comment (MartinMystikJonas) [reply src/Rule/LatteTemplatesRule.php]: or skip completely it is not mandatory

Comment (lulco) [reply src/Rule/LatteTemplatesRule.php]: yes that's what I ment with "nothing"

**Pull request merged**